Assets/Scripts/PlayerScripts/Health.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Health : MonoBehaviour
+{
+    [SerializeField] private int _playerHealth = default;
+    [SerializeField] private int _healthNumber = default;
+    [SerializeField] private Image[] _lifeContainer = default;
```

Comment:
This should be in plural since it is an array of things, maybe `_livesContainers`?